FLEDGE.md

```diff
@@ -924,6 +960,8 @@ The output of `generateBid()` contains the following fields:
     browser to select only some of the returned adComponents in order to help
     make the ad k-anonymous. See [Flexible Component Ad Selection Considering k-Anonymity](#341-flexible-component-ad-selection-considering-k-anonymity)
     for more details.
+*   selectedBuyerAndSellerReportingId: (optional) A string from the interest group's ad's `selectableBuyerAndSellerReportingId` array.  If present and jointly k-anonymous with `buyerAndSellerReportingId`, `buyerReportingId`, the interest group owner, bidding script URL, and render URL, then it will be presented to `reportWin()` and `reportResult()`.
+*   buyerAndSellerReportingIdRequired: (optional) A boolean that when true, indicates this bid should be thrown away if the k-anonymity check on the `selectedBuyerAndSellerReportingId` fails. Setting this to true indicates that reporting of this bid would not operate correctly were the `selectedBuyerAndSellerReportingId` not presented to `reportWin()` and `reportResult()`.
```

Comment:
just to confirm -- assuming this is used to convey a deal bid, this field should always be set to `true` ?

Reply:
If the buyer wants a bid to be contingent on `selectedBuyerAndSellerReportingId` being presented to `reportWin()` and `reportResult()`, then yes, they should set this value to `true` for that bid.  Similarly, if the seller wants the bid's score to be contingent on `selectedBuyerAndSellerReportingId` being presented to `reportWin()` and `reportResult()`, then `scoreAd()` should only score bids when `browserSignals.selectedbuyerAndSellerReportingIdRequired` is `true`.